<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTabelUI</class>
 <widget class="QDialog" name="CreateTabelUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="uiLabelTableName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nume Tabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uiLineEditNumTab">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="uiListCol">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonAddCol">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonRem">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelNumCol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nume Coloana</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uiLineEditNumCol">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelTip">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QComboBox" name="uiComboBoxTip">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>int</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>text</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bool</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>primary key</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="uiLabelSize">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Marimea</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uiLineEditSize">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonFinish">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
